builder/image/templates/builder

```diff
@@ -113,7 +105,7 @@ if [ ! -f Dockerfile ]; then
     BUILD_OPTS+=' deis/slugbuilder'
 
     # build the application and attach to the process
-    JOB=$(eval '$(parse-string "${BUILD_OPTS[@]}")' )
+    JOB=$(${BUILD_OPTS[@]})
```

Comment:
For completeness (because code always explains better than my articulation):

```
><> cat asdf.sh
#!/bin/bash

declare -a arr=('echo' 'hi')

JOB=${arr[@]}
echo $JOB
JOB=`${arr[@]}`
echo $JOB
JOB=$(${arr[@]})
echo $JOB
><> bash asdf.sh
echo hi
hi
hi
```